docker/linux/Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM golang:1.19.3 AS build
-COPY . /go/src/prometheus-nats-exporter
-WORKDIR /go/src/prometheus-nats-exporter
-RUN make prometheus-nats-exporter.docker
-
 FROM alpine:latest as osdeps
+
 RUN apk add --no-cache ca-certificates
 
+ARG BINARY=prometheus-nats-exporter
+
 FROM scratch
```

Comment:
Opinions on using [gcr.io/distroless/static-debian11](https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md) here?  That's essentially what we are making with `scratch`

Reply:
sounds good to me since that includes the certs

Reply:
there's benefit to using same distro - for example @philpennock tracks alpine security updates etc.

Reply:
I think the only things coming from debian in this image are CA Certificates and TZData.  It has some other niceties, such as a `/tmp` directory and an `/etc/passwd` entry though.

Reply:
Clearly this particular image has been working fine without those things though, so fine to just leave it the way it is too